ci/dlang.yml

```
@@ -0,0 +1,16 @@
+name: DLang
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+        
+    steps:
+    - uses: actions/checkout@v1
+    - uses: mihails-strasuns/setup-dlang@v0.2.0
+    - name: Build
+      run: dub build
+    - name: Run tests
+      run: dub test

```

```suggestion
    - name: Build & Test
      run: |
        # Build the project, with its main file included, without unittests
        dub build --compiler=$DC      
        # Build and run tests, as defined by `unittest` configuration
        # In this mode, `mainSourceFile` is excluded and `version (unittest)` are included
        # See https://dub.pm/package-format-json.html#configurations
        dub test --compiler=$DC
```

We might also want to publish artifacts, but that's a bit more involved.